.flowconfig

```diff
@@ -0,0 +1,7 @@
+[ignore]
```

Comment:
I used it do get a better auto completion in intellij. We can kick this out if you want. The cool thing about flow is that it's optional so if you don't configure it in your ide it should not give errors